huey/storage.py

```diff
@@ -122,6 +122,10 @@ def flush_all(self):
 class RedisStorage(BaseStorage):
     def __init__(self, name='huey', blocking=False, read_timeout=1,
                  max_errors=1000, connection_pool=None, **connection_params):
+        redis_url = connection_params.pop('url', None)
+        if redis_url:
+            connection_pool = redis.ConnectionPool.from_url(redis_url, decode_components=True)
```

Comment:
> To fix this: Would you like a ValueError being raised if more than one of these parameters are specified at the same time? I'm happy to supplement this PR :)

That'd be awesome, do you mind? I can handle it as well ifyou prefer.

Making url a regular keyword argument also seems like a good idea to me.